plug_gateway/README.md

```diff
@@ -1,21 +1,9 @@
 # PlugGateway
 
-**TODO: Add description**
-
-## Installation
-
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `plug_gateway` to your list of dependencies in `mix.exs`:
-
-```elixir
-def deps do
-  [
-    {:plug_gateway, "~> 0.1.0"}
-  ]
-end
-```
-
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/plug_gateway](https://hexdocs.pm/plug_gateway).
+A minimally-furnished `Plug` based API gateway so you can see the machinery.
 
+* `mix deps.get` to get your dependencies
+* `mix deps.compile` to compile them
+* `mix compile` to compile your code
```

Comment:
Maybe we don't really need to tell people how to explicitly compile since it just happens anyway when you try to run or test the code. We could replace these with "`iex -S mix` to run your code" maybe.

Reply:
I paste that block everywhere. I should stop, yeah. :)